### PR TITLE
Release 1.4.1

### PR DESCRIPTION
**Pull request by fgiorgetti**

### CODE REVIEW

File: bundle/manifests/1.4.1/skupper-operator.v1.4.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+  - email: skupper@googlegroups.com
+    name: Skupper Community
+  maturity: alpha
+  provider:
+    name: Skupper Project
+    url: https://skupper.io
+  replaces: skupper-operator.v1.3.0
+  skips:
+  - skupper-operator.v1.4.0-rc2
+  - skupper-operator.v1.4.0-rc2
```
Comment (nluaces) [inline bundle/manifests/1.4.1/skupper-operator.v1.4.1.clusterserviceversion.yaml:295]: is this repeated for a reason? should it skip 1.4.0-rc4 as well?

Comment (fgiorgetti) [reply bundle/manifests/1.4.1/skupper-operator.v1.4.1.clusterserviceversion.yaml:295]: Fixing.

File: env.sh [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 # site-controller, service-controller, config-sync, flow-collector
-export SKUPPER_CONTROL_TAG="1.4.0-rc3"
+export SKUPPER_CONTROL_TAG="1.4.1"
 
 # Comma separated list of versions to be skipped (optional)
 # example:
 # SKIP_VERSIONS="1.4.0-rc2,1.4.0-rc3"
 #unset SKIP_VERSIONS
-export SKIP_VERSIONS="1.4.0-rc2"
+export SKIP_VERSIONS="1.4.0-rc2,1.4.0-rc3"
```
Comment (nluaces) [inline env.sh:23]: is it necessary to add 1.4.1-rc4 here as well?

Comment (nluaces) [reply env.sh:23]: there is no 1.4.0-rc4 version in the operator, that's why

**Review by nluaces — APPROVED** [review]

File: env.sh (hunk 2 above)
Comment (fgiorgetti) [reply env.sh:23]: No, because the operator was not released for rc1.

**Pull request merged**